nodejs/docs/sequelize/migrations_and_model_definition.md

```diff
@@ -0,0 +1,111 @@
+# Model definition
+
+The models in Sequelize can be defined functionally or by classes, but we usually use the functional form.
+
+### The `define(model, attributes, [options])` method
+
+To define mappings between a model and a table, use the `define` method. In this case Sequelize will then automatically add the attributes createdAt and updatedAt to it to have control of inserts or updates (you can avoid this attributes with the `timestamps` option in the definition).
+For example:
+
+```javascript
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      mail: { type: DataTypes.STRING },
+      fullName: { type: DataTypes.STRING },
+      phone: { type: DataTypes.STRING }
+    },
+    {
+      timestamps: false
+    }
+  );
+  return User;
+};
+```
+
+Here we are defining a constant called `User` (note that this name is in PascalCase because it is a model) and his value is the result of `sequelize.define` method. After this definition, the function return the User model.
+
+### Options of `define` method
+
+[Here](https://sequelize.readthedocs.io/en/2.0/docs/models-definition/#configuration) you can see the options for the third parameter of `define` method
+
+# Migrations
+
+### How yo create a migration
+
+To create a migration we can use the following command on your project roots:
```

Comment:
Could you add how works migrations, please? (Validations with table `sequelizeMeta` for example)